opty/tests/test_utils.py

```diff
@@ -269,7 +269,7 @@ def test_ufuncify_matrix():
 
     a_vals = np.random.random(n)
     b_vals = np.random.random(n)
-    c_vals = np.random.random(n)
+    c_vals = np.abs(np.random.random(n))
```

Comment:
I still get a warning for a pow() when evaluating. I'm not sure what number is causing that.

Reply:
```
opty/tests/test_utils.py::test_ufuncify_matrix
  /home/moorepants/src/opty/opty/tests/test_utils.py:283: RuntimeWarning: invalid value encountered in power
    result[:, 0, 0] = a_vals**2 * np.cos(np.pi*b_vals)**c_vals

opty/tests/test_utils.py::test_ufuncify_matrix
  <lambdifygenerated-25>:4: RuntimeWarning: invalid value encountered in scalar power
    return array([[x0*cos(pi*b)**_Dummy_250, _Dummy_250**4 + tan(b)/sin(x1)], [-sqrt(_Dummy_250) + b**2 + x0, x1*(_Dummy_250 + x1)*sin(b)/a]])
```